src/search/pdbs/pdb_heuristic.cc

```diff
@@ -33,13 +33,27 @@ int PDBHeuristic::compute_heuristic(const State &ancestor_state) {
     return h;
 }
 
+static basic_string<char> paper_references() {
+    return utils::format_conference_reference(
+        {"Stefan Edelkamp"},
+        "Planning with Pattern Databases",
+        "https://ecp01.icaps-conference.org/papers/A-longpapers/2/edelkamp.pdf",
+        "Proceedings of the Sixth European Conference on Planning (ECP 2001)",
+        "84--90",
```

Comment:
The tools we use don't use LaTeX syntax, so we use a single dash for page ranges, not two dashes.
The linked version of the paper is not the one described by the reference. (The linked version has 12 pages, not 7; its page numbers would be 13-24.) I think the linked version is the pre-proceedings version (in Springer style) from 2001, not the actual "published" version produced in AAAI style much later. Perhaps that's a good compromise if we can't link the actual published paper, but it is confusing.